src/components/Input.js

```diff
@@ -137,12 +137,15 @@ export const Input = props => {
       useNativeDriver: false,
     }).start();
 
-    inputRef.current &&
-      inputRef.current.setNativeProps({
-        style: {
-          top: isFocused ? 10 : 0,
-        },
-      });
+    {
+      !!label &&
+        inputRef.current &&
+        inputRef.current.setNativeProps({
+          style: {
+            top: isFocused ? 10 : 0,
+          },
+        });
+    }
```

Comment:
@Tsudhishnair I think we no need to wrap in curly brackets here, as this code is not in JSX.